lib/api/api_connect.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:http/http.dart' as http;
+import 'dart:async';
+import 'dart:convert';
+import 'package:Solon/api/message.dart';
+import 'package:Solon/api/user.dart';
+import 'package:Solon/api/proposal.dart';
+import 'package:Solon/api/comment.dart';
+import 'package:Solon/api/event.dart';
+import 'package:Solon/api/forumpost.dart';
+import 'package:Solon/api/vote.dart';
+import 'dart:io';
+import 'package:flutter/services.dart';
+import 'package:intl/intl.dart';
+
+class APIConnect {
+  static String _url = "https://api.solonedu.com";
+
+  static Future<String> loadHeader() async {
+    return await rootBundle.loadString('assets/secret');
+  }
+
```

Comment:
lets consider making this a variable so we don't call loadHeader a bunch of times